blazor/dropdown-list/style.md

```diff
@@ -123,4 +145,115 @@ Use the following CSS to customize the appearance of popup element.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/style/html-attributes.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with different font family](./images/style/blazor_dropdown_html-attributes.png)
+
+## Set the various font family for dropdown list elements
+
+You can override using the selector for changing the font-family of dropdown list. Also, you can override using the specific class name through [CssClass](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_CssClass) property.
+
+In the below sample changed the font family of Dropdownlist, ListItem in dropdownlist and filterInput text.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/style/dropdown-font-family.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with different font family](./images/style/blazor_dropdown_font-family.png)
+
+## Show tooltip on list item
+
+You can achieve this behavior by using tooltip component. When the mouse is hovered over the DropDownList option, a tooltip appears with information about the hovered list item.
+
+The following code demonstrates how to display a tooltip when hovering over the DropDownList option.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/style/dropdown-with-tooltip.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with tooltip](./images/style/blazor-dropdownlist-tooltip.png)
+
+## Customize selected item opacity
+
+You can set opacity to the selected item using below selector.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/style/dropdown-with-opacity-style.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with opacity style](./images/style/blazor_dropdown_opacity-style.png)
+
+## Customizing the height
+
+### Height of dropdownlist
+
+You can use [PopupHeight](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_PopupHeight) property to change the height of the popup.
+
+% highlight cshtml %}
+
+{% include_relative code-snippet/style/popup-height.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with Popup height](./images/style/blazor_dropdown_height.png)
+
+### Width of dropdownlist
+
+You can use [PopupWidth](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_PopupWidth) property to change the width of the popup. By default, the popup width sets based on the width of the component. and use [Width](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Width) to change the width of the component.
```

Comment:
Use the [PopupWidth](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_PopupWidth) property to change the width of the popup. By default, the popup width is set based on the component's width. Use the [Width](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Width) to change the width of the component.